### DISCUSSION TITLE
Flow launcher Integration

### DISCUSSION THREAD

**Discussion by CUR5EDFLVME** [luandersonn/MyFTP#46] [Ideas]
Sorry for asking here, I'm kinda new to github. I use the app called Flow Launcher(https://github.com/Flow-Launcher/Flow.Launcher) very often, it helps speeding up doing a lot of different stuff, while using a pc. I would really appreciate if you check out the app, maybe you will like it and then make a plugin to open up a ftp session as fast as possible. 
Someone already made a plugin to run filezilla session immediately outta flow launcher search bar. It works like spotlight, but for windows and better.

I was originally searching for any modern simple ftp to use, because i got kinda tired to use filezilla all the time.
Thank you for your time and thank you for your project!

**Comment by luandersonn** [comment]
Hello @CUR5EDFLVME,

Thanks for the message and for checking out MyFTP. I appreciate your support.

I'd love to learn more about how you're using FileZilla with Flow Launcher. If you could share some examples or details, that would be awesome and really helpful.

Also, just a heads up—I'm currently migrating MyFTP to Windows UI 3.0 for a better experience. If you're interested, you can explore more about this in issue #45.

Thanks again for your time and the great suggestion!

**Comment by CUR5EDFLVME** [comment]
Yo! Thank you for response! Yeah, I found that you're reworking the app, glad to see that, so I got subscribed to this project page.

So, basically, I configured some profiles in filezilla site manager to access different servers, and this flow launcher plugin(https://github.com/shaffe-fr/flow-filezilla) lets me search my profile and open filezilla sessions in a matter of one command. So it could save ton of time, by reducing the amount of clicks you need to do in order to start working with actual files.

I find that the program in general is mad convenient, I use it all the time in all different scenarios.

(there should be a list, but I'm sending this out of my second pc, so there's only one xd, but you've got the point)
<img width="630" alt="Flow Launcher_U70CFtlpAP" src="https://github.com/luandersonn/MyFTP/assets/67831409/1bb1f5eb-7f36-4d92-bafd-041ec456822e">
<img width="961" alt="filezilla_gjb1ETonta" src="https://github.com/luandersonn/MyFTP/assets/67831409/c3117423-4ef6-4276-836d-60c52736c9f8">
<img width="602" alt="QuickLook_QtuVfEF4OY" src="https://github.com/luandersonn/MyFTP/assets/67831409/f9dbe810-8f24-4d74-8c2c-d60f571805ca">
